lib/gtest_1.7.0/src/tls-linux-clang.cpp

```diff
@@ -0,0 +1,6 @@
+// needed on linux when using clang for thread_local storage to work
```

Comment:
Can you include links in here to the stack overflow stuff just for future reference?

Reply:
Sure